src/PharoThings-Hardware-Core.package/PotBoard.class/instance/writeDigitalValue.into..st

```diff
@@ -1,4 +1,4 @@
 operations
 writeDigitalValue: aBit into: aPin
 
-	driver writeDigitalValue: aBit into: aPin
\ No newline at end of file
+	driver writeDigitalValue: aBit into: aPin id
```

Comment:
It is not correct. Driver should decide to use the id of given pin.
It is used for example in dummy driver implementation

Reply:
So fix FirmataDriver instead

Reply:
It works Denis, and the teacher tell's me that it okay.

Reply:
No. It breaks raspberry boards and dummy boards..
Look at implementors #writeDigitalValue:into: to see that pin instance is expected.

FirmataDriver implements it wrongly by expecting id instead of pin object. And it needs to be fixed instead of PotBoard method

Reply:
i will try but it is difficult for me.

Reply:
I can do it myself. But it is just single method fix